plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarService.java

```diff
@@ -110,11 +120,55 @@ public void authorize() {
 	}
 
 	/**
-	 * 
-	 * @return service
+	 * @param min
+	 *            the time at which the calendar starts to check for events
+	 * @param number
+	 *            the number of events it should return
+	 * @return the next events after the min time
 	 */
-	public Calendar getService() {
-		return this.service;
+	public Events getEvents(DateTime min, int number) {
+		Events events = new Events();
+		try {
+			events = this.service.events().list(this.primary).setMaxResults(number).setTimeMin(min)
+					.setOrderBy(this.orderBy).setSingleEvents(true).execute();
+		} catch (IOException e) {
+			this.logger.error(this.errorLogger, e);
```

Comment:
try to log a more specific message for example "getEvents() failed with an IOException for DataTime {}", min.toStringRepresentation()